usb/backend/libusb1.py

```diff
@@ -731,6 +731,13 @@ def get_device_descriptor(self, dev):
         except AttributeError:
             dev_desc.port_numbers = None
 
+        try:
+            dev_desc.speed = self.lib.libusb_get_device_speed(dev.devid)
+            dev_desc.iSpeed = None
+        except Exception:
```

Comment:
Bear in mind that if `libusb_get_device_speed` returns an error, `speed` must be `None`.